kolibri/plugins/device/assets/src/modules/manageContent/actions/taskActions.js

```diff
@@ -48,7 +50,7 @@ export function refreshTaskList(store) {
 }
 
 export function refreshDriveList(store) {
-  return TaskResource.localDrives().then(({ data }) => {
+  return client({ url: urls['kolibri:core:driveinfo-list']() }).then(({ data }) => {
```

Comment:
We appear not to, no. I think we've so far skated by without it, because mostly this URL namespacing is hidden inside the API Resource layer, but some documentation of it would be helpful.